### ISSUE TITLE
Missing .editorconfig file

### ISSUE THREAD

**Issue by TotomInc** [rhyneav/papercss#54]
As I see the PRs, everyone is using its own indentation (spaces/tabs, size) and it sometimes mess the file, PR review become a pain.

**Comment by ummahusla** [comment]
Yep, editorconfig is a crucial thing to use when there are multiple collaborators.

**Comment by TotomInc** [comment]
We should agree on certain points of the editorconfig file such as indent size/type, EOL. @rhyneav

**Comment by ummahusla** [comment]
https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by rhyneav** [comment]
Haha! Was literally thinking of that scene when reading this! 

As for the spaces/tabs debate... Does 2 spaces for HTML, CSS, and LESS files make sense? Keep things consistent. It looks like most of the Less files are currently using 4 spaces (oops). 

What other specifications need to be included in the .editorconfig?

**Comment by TotomInc** [comment]
I agree with the 2 spaces for all the files. Generally a basic `.editorconfig` file looks like that:

```
root = true

[*]
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 2
```

**Comment by Fraham** [comment]
Is this issue closed now that #65 has been merged?

**Comment by TotomInc** [comment]
Yep it should be!
